### DISCUSSION TITLE
how to use plugins

### DISCUSSION THREAD

**Discussion by saarsa** [TimWolla/docker-adminer#100] [Q&A]
Sorry for the lack of knowledge but how I use plugins?
I try this code but I see in login Server only numbers
And I can not connect
The code save in /var/www/html/plugins-enabled/test.php

`<?php
require_once('plugins/login-servers.php');

/** Set supported servers
    * @param array array($domain) or array($domain => $description) or array($category => array())
    * @param string
    */
return new AdminerLoginServers(
    $servers = ["test","test2"],
    $driver = 'server'
);`

And this is the list I see in html 

`tr th Server /th td select name="auth[server]" class="" option 0 /option option 1 /option /select /td /tr`

**Comment by TimWolla** [comment]
>  I try this code but I see in login Server only numbers

Please refer to the plugin's author for support. I am only packaging Adminer for easy use with Docker and I am not knowledgeable about all Adminer plugins. 

I believe you might achieve what you need by using `$servers = ["mysql.example.com" => "My MySQL server"]`.

**Reply by saarsa** [reply]
in $servers from what I understand I need to enter the ip or containername like docker support
But I will check with the plugin's author thanks
